batch7crud-roll4/src/main/webapp/resources/css/style.css

```diff
@@ -0,0 +1,203 @@
+body {
+    background: #ECF0F5;
+}
+
+a {
+    text-decoration: none;
+    font-weight: bold;
+    color: #3C8DCA;
+}
+
+a:hover {
+    color: #72AFD2;
+}
+
+.logo a {
+    color: #fff;
+    font-size: 26px;
+    font-weight: bold;
+    text-decoration: none;
+    display: block;
+}
+
+.logo a:hover {
+    background: #367FAA;
+}
+
+.username {
+    float: left;
+    display: block;
+    padding: 0 10px;
+    color: #fff;
+    font-size: 14px;
+    text-decoration: none;
+    line-height: 50px;
+}
+
+.username:hover {
+    background: #367FAA;
+}
+
+.logout-btn {
+    float: left;
+    padding: 15px 15px;
+}
+
+.logout-btn:hover {
+    float: left;
+    background: #367FAA;
+}
+
+/*navigation style*/
+.navigation ul {
+    list-style: none;
+}
+
+.navigation ul li {
+    display: block;
+    line-height: 50px;
+}
+
+.navigation ul li a {
+    display: block;
+    text-decoration: none;
+    font-weight: bold;
+    color: #444444;
+    font-size: 14px;
+    padding-left: 50px;
+}
+
+.navigation ul li:hover {
+    background-color: #F4F4F5;
+}
+
+li.employees {
+    background: url(../images/employees.png) 10px center no-repeat;
+}
+
+/*section style*/
+.section {
+    width: 90%;
+    margin: 10px 1% 0 1%;
+    background: #fff;
+    padding: 10px;
+}
+
+.section-header {
+    line-height: 40px;
+    border-bottom: 1px solid #F4F4F4;
+    font-size: 16px;
+    margin-bottom: 10px;
+}
+
+.section-footer {
+    line-height: 40px;
+    border-top: 1px solid #F4F4F4;
+    font-size: 16px;
+    margin-top: 10px;
+}
+
+/*table style*/
+.table table {
+    background: #fff;
+    margin: 0 auto;
+    text-align: left;
+    border-collapse: collapse;
+}
+
+.table table tr:nth-child(odd) {
+    background: #F9F9F9;
+}
+
+.table table tr th, td {
+    padding: 10px;
+    border: 1px solid #F4F4F4;
+    font-size: 14px;
+    color: #333339;
+}
+
+/*pagination style*/
+.pagination {
+    text-align: center;
+    padding-top: 15px;
+}
+
+.pagination ul {
+    list-style: none;
+    display: inline-block;
+}
+
+.pagination ul li {
+    float: left;
+    line-height: 40px;
+    padding: 0 10px;
+    border: 1px solid #F4F4F4;
+}
+
+.pagination ul li a {
+    display: block;
+}
+
+.pagination ul li:hover {
+    background: #F9F9F9;
+}
+
+/*form style*/
+
+label {
+    font-size: 14px;
+    line-height: 50px;
+    width: 100px;
+    float: left;
+}
+
+.form {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Check this potential box model size issue [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Abox-model)